.github/workflows/update_v8_snapshot_cache.yml

```diff
@@ -1,10 +1,134 @@
 name: Update V8 Snapshot Cache
-on: [workflow_dispatch]
+on:
+  schedule:
+    # Run every Wednesday at 00:00 UTC
```

Comment:
I chose this so it'll be after the release. I'm ensuring to run from scratch then, so any changes will be caught early rather than right before the release.